services/core-api/app/api/utils/helpers.py

```diff
@@ -1,6 +1,21 @@
 import re
 
+
 def clean_HTML_string(raw_html):
-  cleanr = re.compile('<.*?>')
-  cleantext = re.sub(cleanr, '', raw_html)
-  return cleantext
\ No newline at end of file
+    cleanr = re.compile('<.*?>')
+    cleantext = re.sub(cleanr, '', raw_html)
+    return cleantext
+
+
+"""
+Add sufix to permit number adding one to versioning.
+"""
```

Comment:
Can you update this message to specify that it's for the 'Draft permit Number' only?

Reply:
Sure Meredith, this function is very generic, to be reused in similar cases. I will add further comments on it.